core/templates/domain/state/StateObjectFactory.ts

```diff
@@ -155,8 +162,10 @@ export class StateObjectFactory {
     private paramchangesObject: ParamChangesObjectFactory,
     private writtenTranslationsObject: WrittenTranslationsObjectFactory) {}
 
+  // Type of Constant here doesn't match with the type StateBackendDict,
+  // so we will have to typecast it as unknown first.
```

Comment:
Why is that and can it be solved in some different way?

Reply:
done